src/backend/aspen/workflows/nextstrain_run/export.py

```diff
@@ -97,6 +103,32 @@ def cli(
     print(json.dumps(aligned_gisaid_dump))
 
 
+def dump_yaml_template(
```

Comment:
would this replace this line?: https://github.com/chanzuckerberg/czgenepi/blob/48132693794ca0047b2a4a2b129d5b4854a939b8/src/backend/aspen/workflows/nextstrain_run/run_nextstrain_ondemand.sh#L51-L52

Reply:
I'm not sure what your confusion is, they have very different use cases. `dump_yaml_template()` generates a `builds.yaml` file. It is only useful for local debugging, like in my case I didn't have any sequence or metadata files, but I still wanted to output a `builds.yaml` since we were debugging that. This line you highlighted is an s3 command that copies a `builds.yaml` file to s3.

Reply:
Got it! So the `dump_yaml_template` is just for debugging. The other one is part of a regular run. I'll put it into my wiki. Thanks Lucia <3